code/components/citizen-server-impl/src/state/ServerGameState.cpp

```diff
@@ -4661,6 +4825,11 @@ static std::function<bool()> GetEventHandler(fx::ServerInstanceBase* instance, c
 	bool isReply = buffer.Read<uint8_t>(); // is reply
 	uint16_t eventType = buffer.Read<uint16_t>(); // event ID
 
+	if (Is2060() && eventType > 55) // patch for 1868+ game build as `NETWORK_AUDIO_BARK_EVENT` was added
```

Comment:
I'm not sure if this is the best way to resolve events ordering issue, but you have to make that check to also work with 2189 game build. You can use `xbr::IsGameBuildOrGreater<2060>()`.